[2022-06-03 01:52:49,136] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T01:52:38.880792+00:00 [queued]>
[2022-06-03 01:52:49,159] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T01:52:38.880792+00:00 [queued]>
[2022-06-03 01:52:49,160] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 01:52:49,162] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-03 01:52:49,163] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 01:52:49,202] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-03 01:52:38.880792+00:00
[2022-06-03 01:52:49,212] {standard_task_runner.py:52} INFO - Started process 154 to run task
[2022-06-03 01:52:49,220] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-03T01:52:38.880792+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmpc4vis10f', '--error-file', '/tmp/tmpbs4denv0']
[2022-06-03 01:52:49,223] {standard_task_runner.py:80} INFO - Job 118: Subtask processing_data
[2022-06-03 01:52:49,383] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T01:52:38.880792+00:00 [running]> on host b7655f13eea6
[2022-06-03 01:52:49,538] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T01:52:38.880792+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T01:52:38.880792+00:00
[2022-06-03 01:52:51,013] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-03 01:52:51,015] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-03 01:52:52,954] {python.py:173} INFO - Done. Returned value was: None
[2022-06-03 01:52:52,992] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220603T015238, start_date=20220603T015249, end_date=20220603T015252
[2022-06-03 01:52:53,052] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-03 01:52:53,151] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
